server/models/handlers/appointment.js

```diff
@@ -24,7 +24,18 @@ async function getAppointmentDetails(studentId, workerId, status) {
     return appointmentModel.getAppointmentDetails(studentId, workerId, status);
 }
 
+// Attempts to cancel all the worker appointments and updates their availability to unavailable for the entire day
+// or for a specified period.
+// Returns true if the worker appointments and their times were successfully updated and false if error encountered.
+async function cancelWorkerAppointments(workerId, startTime, endTime) {
```

Comment:
Another endpoint was created instead of leveraging the `api/worker-availability` to capture the unique nature of this request as it is different from that of a worker inputting their availability. For example, the worker does not need to specify the endTime here but they must do so when inputting their availability.